.github/ISSUE_TEMPLATE/issue.yml

```
@@ -0,0 +1,106 @@
+name: Directory Connector Bug Report
+description: File a bug report
+title: "[DC] "
+labels: ["bug"]
+type: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please do not submit feature requests. The [Community Forums](https://community.bitwarden.com) has a section for submitting, voting for, and discussing product feature requests.
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps To Reproduce
+      description: How can we reproduce the behavior.
+      value: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. Click on '...'
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Result
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Result
+      description: A clear and concise description of what is happening.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots or Videos
+      description: If applicable, add screenshots and/or a short video to help explain your problem.
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional Context
+      description: Add any other context about the problem here.
+  - type: checkboxes
+    id: os
+    attributes:
+      label: Operating System
+      description: What operating system are you seeing the problem on?
+      options:
+        - label: Windows
+        - label: macOS
+        - label: Linux
+        - label: Other
+    validations:
+      required: true
+  - type: input
+    id: os-version
+    attributes:
+      label: Operating System Version
+      description: What version of the operating system(s) are you seeing the problem on?
+  - type: checkboxes
+    id: directories
+    attributes:
+      label: Directory Service
+      description: What directory service(s) are you seeing the problem on?
+      options:
+        - label: LDAP or Active Directory
+        - label: Microsoft Entra ID
+        - label: Google Workspace
+        - label: Okta Universal Directory
+        - label: OneLogin
+        - label: Other
+    validations:
+      required: true
+  - type: checkboxes
+    id: application-type
+    attributes:
+      label: Application Type
+      description: Which Directory Connector application are you seeing the problem on?
+      options:
+        - label: GUI
+        - label: CLI
```

I agree with the robot that this could be slightly clearer. e.g.


```suggestion
        - label: GUI (the desktop application)
        - label: CLI (the bwdc command line application)
```